bowling/src/frameParser.cpp

```diff
@@ -0,0 +1,54 @@
+#include "frameParser.hpp"
+
+#include <iostream>
+#include <iterator>
+#include <algorithm>
+
+parsedFrame FrameParser::parse(const std::string& gamePoints) {
+  std::vector<size_t> parsed {};
+  size_t last_parsed_char{};
+  auto tokenized = Tokenizer::splitToFramesAlt(gamePoints);
+  std::transform(tokenized.second.cbegin(),
+		 tokenized.second.cend(),
+		 back_inserter(parsed),
+		 [&last_parsed_char] (char el) {
+		   if (el == 'X') {
+		     last_parsed_char = 10;
```

Comment:
we use camelCase variable convention

Reply:
what exacly last_parsed_char hold?   why not char?  name does not tell us anything

Reply:
last_parser_char holds last calculated "shot " and its use to calculate exact value from "/" -> that is the way that is used in game.cpp

Reply:
Made somewhat easier to read.